protocols/v2/codec-sv2/src/lib.rs

```diff
@@ -72,7 +72,8 @@ impl State {
         }
     }
 
-    pub fn step_2(&mut self, message: [u8; 170]) -> core::result::Result<Self, Error> {
+    // TODO the length of the message should be 170 +32 bytes due to the ellswift encoding
```

Comment:
What do you mean by this TODO? Looks like that's implemented?

Reply:
Ok no this was already solved